examples/6_totp.py

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env python3
+
+###################################
+# WHAT IS IN THIS EXAMPLE?
+#
+# This is a simple TOTP bot that makes use of
+# the team encrypted key-value store, which
+# we interact with using KVStoreClient.
+###################################
+
+import asyncio
+import logging
+import os
+import sys
+from enum import Enum
+
+import pyotp
+
+from pykeybasebot import Bot, EventType
+
+logging.basicConfig(level=logging.DEBUG)
+
+if "win32" in sys.platform:
+    # Windows specific event-loop policy
+    asyncio.set_event_loop_policy(asyncio.WindowsProactorEventLoopPolicy())
+
+
+class TotpMsg(Enum):
+    PROVISION = "provision"
+    REPROVISION = "reprovision"
+    REMOVE = "remove"
+    NOW = "now"
+    URI = "uri"
+    LIST = "list"
+
+
+class TotpHandler:
+    """
+    TotpHandler handles commands sent via chat and uses the team key-value store to
+    provide TOTP client functionality.
+
+    Background: The team encrypted key-value store stores (key, value)
+    pairs, with unique (team, namespace, key)s. The value is encrypted, and the
+    team and namespace are not.
+
+    TotpHandler listens to chat messages of the form:
+
+    `totp {provision|reprovision|remove|now|uri} <issuer>` and
```

Comment:
i think the way we're moving with restricted bots, it might make sense to have this be `!totp...`.

Reply:
the `!` is how restricted and unrestricted bots will eventually register their commands in a channel. so may as well do it here too.